pages/components/Sidebar.js

```diff
@@ -52,6 +52,7 @@ export const Sidebar = ({ onClick }) => (
       <MenuItem onClick={onClick} href="/page-headings" title="Page headings" />
       <MenuItem onClick={onClick} href="/forms" title="Form elements" />
       <MenuItem onClick={onClick} href="/buttons" title="Buttons and links" />
+      <MenuItem onClick={onClick} href="/fast-links" title="Fast links" />
```

Comment:
There is the navigation page above, its currently empty, would it be better to put this in there?

Reply:
Interesting, I hadn't noticed that. Maybe what I'll do is put the link in that page but this is definitely a specific sub-case of navigation

Reply:
You might want to check how I ended up implementing this.